blueprints/ember-cli-textarea-autosize/index.js

```diff
@@ -0,0 +1,8 @@
+/*jshint node:true*/
+module.exports = {
+  description: 'The blueprint that adds the autosize bower library to the project.',
+  normalizeEntityName: function () {},
+  afterInstall: function () {
+    return this.addBowerPackageToProject('autosize');
```

Comment:
In order for the `autosize` library to be available in apps that use this addon, we need to create this blueprint to tell it to grab the library from bower.